ruby-challenges.rb

```diff
@@ -1,34 +1,48 @@
 # ASSESSMENT 4: Ruby Coding Practical Questions
 # MINASWAN ✌️
 
+
+
 # --------------------1) Create a method that takes in a number and determines if the number is even or odd. Use the test variables provided.
+def even_or_odd number
+    number % 2 == 0 ? "#{number} is even" : "#{number} is odd"
```

Comment:
Nice use of ternary operator!  Also, checkout the .even? and .odd methods in Ruby.